### PR TITLE
#1 add deploy task

### PR DESCRIPTION
**Pull request by jdvr**
##Added

- List of remote wars to deploy
- List of local (control machine) wars to deploy
- add two task on a `deploy.yml` file with _deploy_ task

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/tomcat-role/tree/develop)
 
+## [1.0.1](https://github.com/idealista/tomcat-role/tree/1.0.1)
+
+### Added
+- *[#1](https://github.com/idealista/redis-role/issues/1) Add deploy task* @jdvr
```
Comment (dortegau) [inline CHANGELOG.md]: not redis-role :)

File: defaults/main.yml [hunk 2]
```diff
@@ -49,3 +50,11 @@ tomcat_users:
   - name: admin
     password: must-be-changed
     roles: manager-gui
+
+## Deploy
+tomcat_download_wars: true
+tomcat_war_to_deploy_urls:
+  - https://tomcat.apache.org/tomcat-8.5-doc/appdev/sample/sample.war
```
Comment (dortegau) [inline defaults/main.yml]: shouldn't this variable to be in the test folder instead of in the default folder?

**Comment by jdvr** [comment]
The travis build is falling because of a timeout

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/tomcat-role/tree/develop)
 
+## [1.0.1](https://github.com/idealista/tomcat-role/tree/1.0.1)
```
[the remark this answers is not in the document]
Comment (jdvr) [reply CHANGELOG.md]: Good idea, this is not a patch is a new feature

**Pull request merged**